front/src/utils/date.ts

```diff
@@ -241,3 +241,10 @@ export const generateISODateFromDateTime = ({ date, hours, minutes }: ScheduleCo
   date.setMinutes(minutes);
   return formatLocaleDateToIsoDate(date);
 };
+
+/** Format a date to a string 'DD/MM/YY', for instance '01/01/24' */
+export const formatDateString = (date?: Date | null) => {
+  if (!date) return '';
+  const dayjsDate = dayjs(date);
+  return dayjsDate.format('DD/MM/YY');
```

Comment:
why not just ` return dayjs(date).format('DD/MM/YY')` ?

Reply:
Done